tuner.c

```diff
@@ -120,13 +120,12 @@ void cSatipTuner::Action(void)
                break;
           case tsSet:
                debug4("%s: tsSet [device %d]", __PRETTY_FUNCTION__, deviceIdM);
+               Disconnect();
```

Comment:
This will slowdown tuning a bit - would it better to just use a quirk condition check here?

Reply:
Response times of SAT>IP are quite fast. Strictly speaking there is no need to disconnect totally in case of an error. A `TEARDOWN` command should be sufficient. 
But I am not familiar with the plugin code nor with the VDR code so I choose the option with minimal risk. I did not want to mess up the internal state variables. Feel free to use a better implementation. I can check whether it works with FritzOS.

Reply:
Well, just add comment here: `// some servers require a teardown before retuning` or something